algrtm/src/app/customSolutions/page.js

```diff
@@ -1,7 +1,32 @@
+import CustomSolutionsSection from "@/components/CustomSolutionsSection";
+import IconsColoredComponent from "@/components/IconsColoredComponent";
+import LinksSection from "@/components/LinksSection";
+import OfferSection from "@/components/OfferSection";
+import RoundedTextComponent from "@/components/RoundedTextComponent";
+
+import SolutionsSection from "@/components/SolutionsSection";
+import TechnologiesSection from "@/components/TechnologiesSection";
+import RoundedRectangle from "@/components/shared/RoundedRectangle";
 import React from "react";
 
 const CustomSolutions = () => {
-  return <div>Custom SOLUTIONS</div>;
+  return (
+    <main className="flex flex-col items-center justify-center px-[2.75rem]">
```

Comment:
Adding `padding` to the `<main>` element causes a problem with the background color, as the background will not cover the screen from left to right, instead there will be white stripes on the sides in sections with a black background:

![image](https://github.com/Nospoko/algrtm/assets/26410548/bf0997b5-addb-461d-9d66-88e4db6cee27)

In Figma, the dark background color is displayed from left to right, so removing the `padding` will fix this effect